src/main/java/org/skriptlang/skript/lang/arithmetic/Arithmetics.java

```diff
@@ -87,7 +87,7 @@ public static boolean operationExists(Operator operator, Class<?> leftClass, Cla
 	@SuppressWarnings("unchecked")
 	public static <L, R, T> OperationInfo<L, R, T> getOperationInfo(Operator operator, Class<L> leftClass, Class<R> rightClass, Class<T> returnType) {
 		OperationInfo<L, R, ?> info =  getOperationInfo(operator, leftClass, rightClass);
-		if (info != null && returnType.isAssignableFrom(info.getReturnType()))
+		if (info != null && (returnType.isAssignableFrom(info.getReturnType()) || info.getReturnType().isAssignableFrom(returnType)))
```

Comment:
why this change?

Reply:
That's not safe to do. If i'm asking for a operation that returns Long, I don't want to get one that returns Number. That could be returning Doubles, which is not what I wanted.